ipn/ipnlocal/local.go

```diff
@@ -360,7 +364,7 @@ func NewLocalBackend(logf logger.Logf, logID logid.PublicID, sys *tsd.System, lo
 
 type componentLogState struct {
 	until time.Time
-	timer *time.Timer // if non-nil, the AfterFunc to disable it
+	timer tstime.TimerController // if non-nil, the AfterFunc to disable it
```

Comment:
comment is no longer right?

Reply:
No still is the same - tstime.TimerController wraps time.Timer behavior